views/widget.scout.admin.php

```diff
@@ -20,30 +20,51 @@
 			$api_secret = $sailthru['sailthru_api_secret'];
 			$client = new WP_Sailthru_Client( $api_key, $api_secret );
 
-			// Get Sections
-			$sections = $client->apiGet('section');
 
-			// get sections
-			if (is_array ( $sections )) {
+			// Check the user has access to SPM. 
+			$settings = $client->apiGet('settings');
+			// set to be disabled by default for a safe fallback
+			$spm_enabled = false;
 
```

Comment:
So this is admin UI? What happens if it can't connect to Sailthru? You also shouldn't need to set a default for spm_enabled. The check for spm_enabled should properly return a boolean.